lib/tasks/sphinx.rake

```diff
@@ -23,7 +23,7 @@ namespace :sphinx do
       # As we enqueue, we only need the :id
       scope.select(:id).except(:includes).find_in_batches(batch_size: 1000) do |batch|
         batch.each do |record|
-          SphinxIndexationWorker.perform_later(record.class, record.id)
+          SphinxLowPrioIndexationWorker.perform_later(record.class, record.id)
```

Comment:
I think it's better to force queue like 

```ruby
Sidekiq::Client.push({
  'class' => SphinxIndexationWorker,
  'queue' => 'whatever_queue',
  'args'  => [record.class, record.id]
})
```

Instead of creating a new duplicate class.

Reply:
Why not update `SphinxIndexationWorker`?

Reply:
@josemigallas See the description. I want to separate the two things: indexing done in real time on model updates, and full reindexing (that can be stopped, rerun, etc, without causing much problems)

Reply:
@akostadinov I think it might actually be useful to distinguish the two jobs, as the low-prio one can potentially be "cancellable", while the "standard one" not (useful unless they end up running in the same queue at some point)

Reply:
But if we go the Sidekiq client route, we can as well do a [bulk push](https://github.com/mperham/sidekiq/blob/main/lib/sidekiq/client.rb#L97). What do you think?

Reply:
@akostadinov We can't use this Sidekiq interface, because it's based on ActiveJob, not Sidekiq::Worker.
We can refactor it, but it's not trivial and would require changes in `SphinxIndexationWorker`, mainly due to the differences in arguments serialization between ActiveJob and direct Sidekiq.

Do we think we should do this?

Using Sidekiq's `bulk_push` would be nice in this case (and would only be possible using Sidekiq directly, not through ActiveJob), but on the other hand the suggested implementation is simpler and clearer.

Reply:
bulk push sounds faster. I dont know the actual implementation but it may reduce umber of DB operations. I'd go for it if it works.

Reply:
https://github.com/mperham/sidekiq/wiki/Bulk-Queueing

```ruby
    # push 1000 jobs in one network call to Redis, saves 999 round trips
    Sidekiq::Client.push_bulk('class' => SomeJob, 'args' => array_of_args)
```

Reply:
wait, it is activejob job, not sidekiq. No we shouldn't change it.

Reply:
How about using the `enqueue method instead then?

https://api.rubyonrails.org/classes/ActiveJob/Enqueuing.html

Reply:
I think the above should do the job. In theory we should be able to create wrapper sidekiq jobs from the activejob ones. But it's messing up too much with internals and doesn't seem necessary in our situation.

Reply:
@akostadinov thanks for the tip! The `.enqueue` method worked well :+1: